src/tasks/actions/creators.ts

```diff
@@ -141,3 +143,9 @@ export const setLogs = (logs: LogEvent[]) =>
     type: SET_LOGS,
     logs,
   } as const)
+
+export const setCurrentTasksPage = (page: string) =>
```

Comment:
```ts
(page: string)
```
we currently only accept two types of pages right? instead of making this a type `string` which can accept any string value (e.g. `"foo"`, `""`, `"3234"`), if we make it a [TypeScript String Enum](https://www.typescriptlang.org/docs/handbook/enums.html#string-enums) we can list all the possible values and get warned when one isn't used:

```ts
export enum TaskPage {
  TasksPage = 'TasksPage',
  TaskRunsPage = 'TaskRunsPage'
}

export const setCurrentTasksPage = (page: TaskPage) =>
```

Reply:
thank you for including the documentation!